R/z_pages.R

```diff
@@ -99,7 +99,15 @@ initial_commit <- function(tmp_dir, repo) {
   writeLines("## New animint visualization", readme_file_path)
   gert::git_add("README.md", repo = repo)
   gert::git_commit("Initial commit", repo = repo)
-  gert::git_branch_move(branch = "master", new_branch = "main", repo = repo)
+  all_branches <- gert::git_branch(repo)
+  print(all_branches)
+  if (is.data.frame(all_branches)) {
+    current_master <- all_branches$name[all_branches$active]
+  } else {
+    current_master <- all_branches
+  }
+  print(current_master)
+  gert::git_branch_move(branch = current_master, new_branch = "main", repo = repo)
```

Comment:
I tried this new code on my ubuntu system, and I am getting this error:
```r
> animint2pages(viz, github_repo = "my_animint2_plots", commit_message = "New animint", private = TRUE)
Le chargement a nécessité le package : gert
Le chargement a nécessité le package : gh
Looking up https credentials for https://github.com/tdhock/my_animint2_plots.git
[1] "main"
[1] "main"
Erreur dans libgit2::git_branch_move : 
  failed to write reference 'refs/heads/main': a reference with that name already exists.
> traceback()
5: stop(e)
4: raise_libgit2_error(-4L, "failed to write reference 'refs/heads/main': a reference with that name already exists.", 
       "git_branch_move", 4L)
3: gert::git_branch_move(branch = current_master, new_branch = "main", 
       repo = repo) at z_pages.R#110
2: initial_commit(tmp_dir, repo) at z_pages.R#84
1: animint2pages(viz, github_repo = "my_animint2_plots", commit_message = "New animint", 
       private = TRUE)
```

Reply:
Thank you for the timely testing and feedback! I've updated the code, let's see if it works this time.